test/twofishes/client_test.rb

```diff
@@ -3,35 +3,81 @@
 module Twofishes
   class Zurich
     def self.geocode(location)
-      YAML::load_file('test/fixtures/response.yaml')
+      if location.responseIncludes == [ResponseIncludes::PARENTS]
```

Comment:
I agree it fees overly complicated... but I felt we should test not only that 'geocode' was called, but that the relevant parameters were passed and reflected in the response.